visualization_msgs/msg/Marker.msg

```diff
@@ -12,6 +12,7 @@ uint8 POINTS=8
 uint8 TEXT_VIEW_FACING=9
 uint8 MESH_RESOURCE=10
 uint8 TRIANGLE_LIST=11
+uint8 ARROW_STRIP=12
```

Comment:
Would it make more sense to call this a DIRECTED_LINE_STRIP as that's an already determined thing and we're just adding direction. 

There's a few places below that reference LINE_STRIP that should be augmented.

Reply:
The geometry shares some functionality with `ARROW`, most notably the appearance and semantics of the `scale` field. I think `ARROW_STRIP` makes this connection more explicit, and conveys the same information as `DIRECTED_LINE_STRIP` in fewer characters.